js-functions/js-functions_basic-functions/js/index.js

```diff
@@ -36,6 +43,16 @@ Then log all variables to the console again.
  - Now increase the number of sales 2 more times and log the data after each increase.
 */
 
-// --v-- write your code here --v--
+function logBookData() {
```

Comment:
You can also give sales as a parameter so that you don't have to redefine sales variable every time.

Reply:
thanks, I will keep this in mind!